app/backplane/Makefile

```diff
@@ -1,23 +1,20 @@
+include app/backplane/sensor_module/Makefile
+
 power:
-	west build -b power_module@2 app/backplane/power_module -p auto -DOVERLAY_CONFIG=debug.conf
```

Comment:
If we don;t have a debug.conf, this is the exact same as power-rel. Do we need both? Same for radio and radio-rel

Reply:
Oops going to be fixed